tools/protoc-gen-mypy.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python -u  "$PWD"/../tools/protoc-gen-mypy.py
```

Comment:
But it is still python not python3?
You should always use the python executable which was used for running setup.py to run this script. 
In the cases of setup.py was not used, the path of the  python executable should be got from the find_package calls.

Reply:
Thank you @snnn for bringing this up.
Yes, I thought of this, but originally the interface of using `protoc-gen-mypy` here is fixed. The only way is to call a script for `protoc-gen-mypy` so I don't know how to use `PYTHON_EXECUTABLE` from CMake for this (Could we call a CMake environment variable from an outside script?)

Otherwise, since onnx won't support python2 since ONNX 1.8, could we just use python3 here?